next.config.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
     HUMAN_ID_GET_WEB_LOGIN_URL: process.env.HUMAN_ID_GET_WEB_LOGIN_URL,
     HUMAN_ID_CLIENT_ID: process.env.HUMAN_ID_CLIENT_ID,
     HUMAN_ID_CLIENT_SECRET: process.env.HUMAN_ID_CLIENT_SECRET,
+    REDIRECT_MAIN_URL: process.env.REDIRECT_MAIN_URL || '',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new environment variable `REDIRECT_MAIN_URL` has been added with a default value of an empty string. If this environment variable is critical for some functionality, consider adding a fallback or default value other than an empty string. An empty string might lead to unexpected behavior if the environment variable is not set.
```diff
- REDIRECT_MAIN_URL: process.env.REDIRECT_MAIN_URL || '',
+ REDIRECT_MAIN_URL: process.env.REDIRECT_MAIN_URL || 'default_value',
```
Remember to replace `'default_value'` with an appropriate value based on your application's requirements.


<!-- This is an auto-generated comment by OSS CodeRabbit -->